caravel/models.py

```diff
@@ -412,6 +412,8 @@ class Database(Model, AuditMixinNullable):
     cache_timeout = Column(Integer)
     select_as_create_table_as = Column(Boolean, default=False)
     expose_in_sqllab = Column(Boolean, default=False)
+    allow_run_sync = Column(Boolean, default=True)
```

Comment:
is there ever a case where these would both be true? is there a preferred default? if so maybe we only need one column here.